cloud-optimized-geotiffs/cogs-details.qmd

```diff
@@ -17,9 +19,35 @@ Scenario: If the COG is intended only for visualization, conversion to 3 band by
+
+Some other compression algorithms may be preferred depending on the data type and distribution, and if the goal is maximum compression or not. Codecs that produce the smallest file sizes _tend_ to take longer to read into memory. If the network bandwidth to load the file is slow, then a very efficient compression algorithm may be most efficient, even if it takes longer to decompress when loaded. Alternatively, if the network speed is very fast, or if reading from a local disk, a slightly less efficient compression codec that decompresses faster may be preferred.
+
+There are many posts on the internet exploring GeoTIFF compression and performance:
+
+- [Guide to GeoTIFF compression and optimization with GDAL](https://kokoalberti.com/articles/geotiff-compression-optimization-guide/)
+- [GeoTiff Compression for Dummies](http://blog.cleverelephant.ca/2015/02/geotiff-compression-for-dummies.html)
+- [GeoTiff compression comparison](https://digital-geography.com/geotiff-compression-comparison/)
+
+### Internal compression
+
+GeoTIFFs have compression internal to the file, meaning that the internal blocks in a GeoTIFF are already compressed. This internal compression is especially useful for COGs, compared to external compression (such as saving a COG inside a ZIP file), since a COG reader can decompress only the specific portion of the file requested, instead of needing to decompress the entire file.
+
+The internal compression of a GeoTIFF also means that it does not need additional compression, and indeed that additional compression will decrease performance. Gzip or ZIP compression applied to a GeoTIFF with internal compression will not make the file smaller.
 
-The biggest benefit to compression is on the storage side. It’s always recommended to use a lossless compression method unless the COG is intended to be used only for visualization. **Deflate** or **LZW** are the recommended compression algorithms for general use. **LERC** (Limited Error Raster Compression) provides very efficient data compression for float data if you're ok rounding to some precision you provide. (This could be useful, e.g. for elevation data where the source data is known to not have precision beyond a known value.) Some other compression algorithms may be preferred depending on the data type and distribution, and if the goal is maximum compression or not. Maximum compression does result in some performance loss.
+It is possible but not recommended to create GeoTIFFs with no internal compression.
```

Comment:
I think it's fine to say `COGs` instead of `GeoTIFFs` here

```
It is possible to create COG with no internal compression, but it's not recommended.

Reply:
I would say both, so either COG (incl. GeoTIFF) or GeoTIFF (incl. COG)

Reply:
Updated to say `COGs or GeoTIFFs`